src/Essentials/src/Geolocation/LocationEventArgs.shared.cs

```diff
@@ -0,0 +1,18 @@
+﻿#nullable enable
+using System;
+
+namespace Microsoft.Maui.Devices.Sensors
+{
+	public class LocationEventArgs : EventArgs
```

Comment:
This probably should match the event name, so maybe `LocationChangedEventArgs`

Reply:
I would rename the class when we decided on the event name and would name it `GeolocationLocationChangedEventArgs` or `GeolocationLocationupdatedEventArgs`, so that all .cs files start with `Geolocation`.

Reply:
I think `GeolocationLocationChangedEventArgs` is good.

Reply:
Done!